tomviz/ModuleManager.cxx

```diff
@@ -665,8 +676,17 @@ void ModuleManager::onPVStateLoaded(vtkPVXMLElement*,
         options["pvXml"] = dsObject["pvReaderXml"];
       }
       QString fileName = dsObject["fileName"].toString();
+      // If we have a fileName make it absolute
+      if (!fileName.isEmpty()) {
+        fileName = d->dir.absoluteFilePath(fileName);
+      }
+
       auto dataSource = LoadDataReaction::loadData(fileName, options);
       dataSource->deserialize(dsObject);
+      if (fileName.isEmpty()) {
```

Comment:
I loaded a dataset (a TIFF if it matters), saved the state file next to it and exited tomviz.  Then I renamed the folder they were in and tried reloading the state file.  I'm getting a segfault in `LoadDataReaction::loadData` just above here.

Reply:
@mathturtle So it looks like the fileName is also being stored in the "pvReaderXml" property in the case of a ParaView reader ( I tested with EMD, which doesn't show this problem ).

Reply:
@mathturtle I have pushed a commit that should fix this up.